src/modules/rbd/rbd-list/rbd-list.js

```diff
@@ -18,6 +18,14 @@
             clusterObj;
 
         vm.createRbd = createRbd;
+        vm.onOpenRbdResizeModal = onOpenRbdResizeModal;
+        vm.resizeRBD = resizeRBD;
+        vm.viewTaskProgress = viewTaskProgress;
+        vm.resizeRBD = resizeRBD;
+        vm.resizeRbd = {"unit": "" , "size": 0};
+        vm.sizeUnits = ["bytes", "KB", "MB", "GB", "TB"];
```

Comment:
@kamleshverma1  As per my conversation with Ju, the size units should be "GB" and "TB". Does Ju has mentioned this change for size units.